frontend/syntaxmeets/src/components/Home/Home.js

```diff
@@ -233,6 +289,37 @@ const Home = (props) => {
                   <br />
                   <br />
 
+
+                      <Row>
+                        <MUIButton
+                          style={{
+                            marginLeft: "auto",
+                            marginRight: "auto",
+                            backgroundColor: "#ffd500",
+                          }}
+                          variant="contained"
+                          size="large"
+                          disabled={disabledName || disabledRoomId}
+                          component={Link}
+                          to={{
+                            pathname: joinRoomId,
```

Comment:
This is be props.roomId, please refer to the updated redux code.

Reply:
Ok I'll go through all the changes.

Reply:
> Ok I'll go through all the changes.

Yes please 🙌 , also if you wish you can create a new clean if if you wish 🙌

Reply:
yes